### PR TITLE
Defining a default environment, responding a message on /.

### PR DESCRIPTION
**Pull request by gfviegas**
-> Defined a default evironment, since there could be users who dont have a node env defined. Setted "dev" as default.

-> Created a hello.js to response a / request. That'll be the first request a new user will make, so he can be sure things are fine before checking the router.

### CODE REVIEW

File: generators/app/templates/hello.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var router = require('express').Router();
+
+router.use('/', function(req,res){
+	res.end("Hello world! Everything is running..");
```
Comment (vinicius0026) [inline generators/app/templates/hello.js]: Please, fix code style issues before merging.

File: generators/app/templates/routes.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
 'use strict';
 
 module.exports = function (app) {
+	app.use('/', require('./hello'));
```
Comment (vinicius0026) [inline generators/app/templates/routes.js]: Please fix code style issues before merging.

**Comment by vinicius0026** [comment]
Hi,
Thanks again for the PRs. I liked this one particularly.
There are some code style issues that must be corrected before merging. I commented directly on them.
Could you fix them?

**Comment by gfviegas** [comment]
Sure, im commiting them by tomorrow and probably pulling some other requests. Great generator btw, i was kinda moving up to building mine but this one has matched my needs.

**Comment by gfviegas** [comment]
I've fixed it. Somehow my IDE defined my identation as TAB but your style is using 3 spaces, am I right?

**Comment by vinicius0026** [comment]
4 spaces

**Comment by vinicius0026** [comment]
Yes, everything alright!

**Pull request merged**